### ISSUE TITLE
Improve stacked bar charts

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/pressfreedomtracker.us#2276]
After some discussions with @ryanwhitney  related to the chart in the unconfirmed incidents page, we found out there are some issues/improvements that might be needed across all Stack bar chat and maybe even every chart.

- Add legend/key instead of buttons ([design](https://www.figma.com/design/agOk78n9QpCcNCjqiZ6ObO/Press-Freedom-Tracker?node-id=309-10371&t=yKPnoCBz2uXAzDEj-4))
- Bars should be selectable as a whole here, with the tooltip showing the two data points for the bar/day.
- Remove <title> attributes as they compete with tooltips. (I believe they already have aria-label as I made a similar change in Increase granularity on homepage visualizations  #2206) [ _This might be something we need to update in all charts?_ ]
- Look into accessibility of stacked bars that don't share any relationship to indicate that they are stacks and not individual bars

I think these are mostly the things we discussed. There might be more improvements that we should make.

**Comment by ryanwhitney** [comment]
Another: In the tooltip, the color to the left of the data should match the color of the bar segment. Right now it's always peach.

(We do this on the homepage vizzes)
